sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -54,13 +54,8 @@ private[hive] case object NativePlaceholder extends Command
  */
 case class DescribeCommand(
     table: LogicalPlan,
-    isExtended: Boolean) extends Command {
-  override def output = Seq(
-    // Column names are based on Hive.
-    AttributeReference("col_name", StringType, nullable = false)(),
-    AttributeReference("data_type", StringType, nullable = false)(),
-    AttributeReference("comment", StringType, nullable = false)())
-}
+    override val output: Seq[Attribute],
```

Comment:
yeah, if we need every operator can be queried like a table, we need make it as a default argument, but does it break the current api or interface ? Shall we do it in another PR or file a separated JIAR?